frontend/src/lib/components/apps/components/table/AppTable.svelte

```diff
@@ -55,6 +55,18 @@
 		}
 	}
 
+	let mounted = false
+	onMount(() => {
+		mounted = true
+	})
+
+	$: Array.isArray(result) &&
+		result.length > 0 &&
+		selectedRowIndex === -1 &&
```

Comment:
you want the selectedRowIndex to be the first condition since this is gonna tick super often and integer comparison is super quick

Reply:
done